de_web_plugin.cpp

```diff
@@ -9564,20 +9466,6 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
                                     enqueueEvent(e);
                                 }
 
-                                if (i->mustRead(WRITE_USERTEST))
-                                {
-                                    ResourceItem *item = i->item(RConfigPending);
-                                    if (item)
-                                    {
-                                        quint16 mask = item->toNumber();
-                                        mask &= ~R_PENDING_USERTEST;
-                                        item->setValue(mask);
-                                        Event e(RSensors, RConfigPending, i->id(), item);
-                                        enqueueEvent(e);
-                                    }
-                                    i->clearRead(WRITE_USERTEST);
-                                }
-
```

Comment:
In my view, the whole code for attributes 0x0032 - 0x0034 can be deleted here. I mean if all Philips devices get their own DDFs, there's probably no reason to keep it? Not sure about the lights though.

Reply:
I think you're right.  I'll do a search and destroy for any code handling these attibutes.
I haven't seen any Hue lights using these attributes.